wpeverest/user-registration
# UR-458 Dev - Email domain denyist

### All Submissions:






### Changes proposed in this Pull Request:
Previously we only had a whitelist email domain feature in pro and it was only available in global settings under the Misc tab but now users will have the option to either allow or deny particular domains in individual forms under User Email fields.

### How to test the changes in this Pull Request:

1. Check backward compatibility.
2. Verify if allowed and denied both are working properly in the registration form and edit profile form.

### Types of changes:




### Other information:




### Changelog entry

>  Dev - Email domain denyist
